watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/amo_1690/amo_1690_ui.py

```diff
@@ -260,17 +260,6 @@ def export_variables(flowsheet=None, exports=None):
         input_category="Membrane evaporator",
         is_output=False,
     )
-    exports.add(
```

Comment:
I guess I should have asked here: this component was duplicated, and since the errors aren't suppressed the tests caught that.

Reply:
The same fs component was being added twice. This by design should trigger an error when `add()` is called a second time with the same `obj`, but the error was being ignored by the unit tests because of the `_log.error()`-only handling.